android/assets/jsons/Tutorials.json

```diff
@@ -168,5 +168,53 @@
+        "Based on this, you can get a feel for which religions have a lot of pressure build up in the city, and which have almost none.",
+        "The city follows a religion if a majority of its population follows the city, and will only than be effected by Follower and Pantheon beliefs of that religion.",
+        "",
+        "Spreading religion happens naturally, but can be sped up using missionaries or great prophets.",
+        "Missionaries can be bought in cities following a major religion, and take the religion of that city.",
+        "So do take care where you are buying them! If another civilization has converted one of your cities to their religion, missionaries bought their will follow their religion.",
+        "Great prophets always have your religion, regardless if they are bought or spawned naturally, but captured great prophets do retain their original religion.",
+        "Both great prophets and missionaries are able to spread religion to cities when they are inside their borders, even cities of other civilizations.",
+        "You can even enter tiles of civilizations with whom you don't have an open borders agreement!",
+        "But do take care, missionaries will lose 250 religious strength each turn they inside foreign lands.",
+        "This diminishes their effectiveness when spreading religion, and if their religious strength ever reaches 0, they have lost their faith and disappear.",
+        "When you do spread your religion, the religious strength of the unit is added as pressure for that religion.",
+        "",
+        "Cities also passively add pressure of their majority religion to nearby cities.",
+        "Each city provides +6 pressure per turn to all cities within 10 tiles, though the exact amount of pressure depends on the game speed.",
+        "This pressure can also be seen in the city screen, and gives you an idea of how religions in your cities will evolve if you don't do anything.",
+        "Holy cities also provide +30 pressure of the religion founded there to themselves, making it very difficult to effectively convert a holy city.",
+        "Lastly, before founding a religion, new cities you settle will start with 200 pressure for your pantheon.",
+        "This way, all your cities will starting following your pantheon as long as you haven't founded a religion yet.",
+        "",
+        "Inquisitors are the last religious unit, and their strength is removing other religions.",
+        "They can remove all other religions from one of your own cities, removing any pressures build up and reclaiming the city for your religion.",
+        "Great prophets also have this ability, removing all other religions when spreading their religion.",
+        "Additionally, when an inquisitor is stationed in or directly next to a city center, units of other religions cannot spread their faith there."
```

Comment:
I wonder - Only units? Or is nearby city pressure muted too? (I mean I could look up the code, but maybe other readers might also wonder at this point)